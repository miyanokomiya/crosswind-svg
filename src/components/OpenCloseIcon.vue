<template>
  <svg
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <circle cx="100" cy="100" r="100" fill="red" />
    <rect x="130" y="130" width="300" height="200" fill="blue" />
  </svg>
</template>

<script lang="ts">
import Vue from "vue";
export default Vue.extend({});
</script>
