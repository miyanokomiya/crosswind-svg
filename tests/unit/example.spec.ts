import { mount } from "@vue/test-utils";
import OpenCloseIcon from "@/components/OpenCloseIcon.vue";

describe("OpenCloseIcon.vue", () => {
  it("state:open", () => {
    const wrapper = mount(OpenCloseIcon);
    expect(wrapper.element).toMatchSnapshot();
  });
});
