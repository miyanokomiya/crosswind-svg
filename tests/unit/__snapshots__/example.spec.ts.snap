// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpenCloseIcon.vue state:open 1`] = `
<svg
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
>
  <circle
    cx="100"
    cy="100"
    fill="red"
    r="100"
  />
   
  <rect
    fill="blue"
    height="200"
    width="300"
    x="130"
    y="130"
  />
</svg>
`;
